src/user/contributing.rst

```diff
@@ -48,5 +48,16 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
    your changes.
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
+   
+  
+.. _Guidelines:
+
+Guidelines
+===========================
+1. For submitting or updating conda-build documentation, follow these rules.
```

Comment:
Here we would change it to a sentence bullet points starts after this. So it would be "For submitting or updating conda-build documentation, follow these rules : "